modules/wallet/scan.go

```diff
@@ -144,8 +155,9 @@ func (s *seedScanner) scan(cs modules.ConsensusSet) error {
 		if err := cs.ConsensusSetSubscribe(s, modules.ConsensusChangeBeginning); err != nil {
 			return err
 		}
+		cs.Unsubscribe(s)
 		if s.largestIndexSeen < s.numKeys()/2 {
-			cs.Unsubscribe(s)
+			s.log.Println("Seed scanning completed, highest key index found at", s.largestIndexSeen)
```

Comment:
We already print a log message like this in `InitFromSeed` and `LoadSeed`. Also, sometimes you don't care about the largest index, as in `SweepSeed`.